src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/LogMailSender.java

```diff
@@ -8,8 +9,21 @@
 @Profile({"dev", "test"})
 @Slf4j
 public class LogMailSender implements MailSender {
+
+    private final MailPostAction mailPostAction;
+
+    @Autowired
+    public LogMailSender(MailPostAction mailPostAction) {
```

Comment:
Hand-crafted constructor can be replaced with Lombok's class level `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` annotation.